src/ruuvitag_ble/parser.py

```diff
@@ -23,14 +25,25 @@ def _start_update(self, service_info: BluetoothServiceInfo) -> None:
             return None
 
         data_format = raw_data[0]
-        if data_format != 0x05:
+        if data_format not in (0x03, 0x05):
             _LOGGER.debug("Data format not supported: %s", raw_data)
             return
 
-        decoder = DataFormat5Decoder(raw_data)
+        decoder_classes: dict[
+            int,
+            Type[Union[DataFormat3Decoder, DataFormat5Decoder]],
+        ] = {
+            0x03: DataFormat3Decoder,
+            0x05: DataFormat5Decoder,
+        }
```

Comment:
This dict should be defined on the module level, it doesn't need to be recomputed everytime we try to decode a thing.